### ISSUE TITLE
[BUG] stats page doesn't display chart

### ISSUE THREAD

**Issue by osaguild** [dev-protocol/devprotocol.xyz#223]
### Problem subject

chart doesn't work on [stats page](https://devprotocol-bjfzuo6g8-devprtcl.vercel.app/stats). image is below.
![image](https://user-images.githubusercontent.com/33506754/191635465-61a465c9-7333-4aa7-ab80-2edd4c564509.png)

### Description

In order to render a chart using d3.js, client rendering is specified for the chart component. like this `<Chart client:load />.`. But Astro's client rendering is not supported from ver 1.0 and chart is not display.
Error message is below
```
A PostCSS plugin did not pass the `from` option to `postcss.parse`. This may cause imported assets to be incorrectly transformed. If you've recently added a PostCSS plugin that raised this warning, please contact the package author to fix the issue.
You are attempting to render <Chart client:load />, but Chart is an Astro component. Astro components do not render in the client and should not have a hydration directive. Please use a framework component for client rendering.
```

### Expected behavior/contents

correct image is [here](https://user-images.githubusercontent.com/33506754/174722271-e63cbf64-ae8f-48dc-b31f-734693017810.png)

### Actual behavior/contents

doesn't show chart

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aggre** [comment]
I think the quickest solution is to implement it as a Vue component instead of an Astro component.

**Comment by osaguild** [comment]
I think so

**Comment by gptkrsh** [comment]
Fixed by #226 (for reference)
